src/App.vue

```diff
@@ -2,26 +2,68 @@
 	<v-app>
 		<app-bar logo="header-logo-white.png" :links="headerLinks"></app-bar>
 		<v-main>
-			<landing-section-component title="Provided Page Title" subtitle="Provided Page SubTitle"> This content is filled via a default slot </landing-section-component>
+			<landing-section-component title="Provided Page Title" background-image="https://i.imgur.com/bX2Uk7C.jpg">
+				<template v-slot:transition-top>
+					<component :is="transitionTop"></component>
+				</template>
+				<template v-slot:transition-bottom>
+					<component :is="transitionBottom"></component>
+				</template>
+			</landing-section-component>
+			<div style="height: 400px;"></div>
+			<section-image background-image="https://i.imgur.com/bX2Uk7C.jpg">
+				<template v-slot:transition-top>
+					<component :is="transitionTop"></component>
+				</template>
+				<template v-slot:transition-bottom>
+					<component :is="transitionBottom"></component>
+				</template>
+			</section-image>
+			<div style="height: 400px;">
+				<v-row>
+					<v-col cols="2"> </v-col>
+					<v-col cols="8">
+						<v-row>
+							<v-col cols="4">
+								<card-info-minimal title="Title" background-image="/card-accommodation.jpg" link=""> </card-info-minimal>
+							</v-col>
+							<v-col cols="4">
+								<card-info-minimal title="Accommodation" background-image="/card-accommodation.jpg" link=""> </card-info-minimal>
+							</v-col>
+							<v-col cols="4">
+								<card-info-minimal title="Title" background-image="/card-accommodation.jpg" link=""> </card-info-minimal>
+							</v-col>
+						</v-row>
+					</v-col>
+				</v-row>
+			</div>
 		</v-main>
-		<!-- <footer-alpha />-->
 	</v-app>
 </template>
 
 <script lang="ts">
 import Vue from 'vue';
 import LandingSectionComponent from '@/core/components/LandingSectionComponent.vue';
+import SectionImage from '@/core/components/SectionImage.vue';
+import SectionTransitionBottomSplash from '@/core/components/transitions/SectionTransitionBottomSplash.vue';
+import SectionTransitionTopSplash from '@/core/components/transitions/SectionTransitionTopSplash.vue';
+import CardInfoMinimal from '@/core/components/cards/CardInfoMinimal.vue';
 import AppBar from '@/core/components/AppBar.vue';
 import Link from '@/core/models/link';
 
 export default Vue.extend({
 	components: {
-		AppBar,
-		LandingSectionComponent
-		// FooterAlpha
+        AppBar,
+		LandingSectionComponent,
```

Comment:
Just me, or is there some weird spacing here?

Reply:
Hmm looks fine on my IDE.